templates/blogs/details.rs.html

```diff
@@ -0,0 +1,49 @@
+@use plume_models::blogs::Blog;
+@use plume_models::posts::Post;
+@use plume_models::users::User;
+@use templates::{base, partials::post_card};
+@use template_utils::*;
+
+@(ctx: BaseContext, blog: Blog, fqn: String, authors: &Vec<User>, total_articles: usize, page: i32, n_pages: i32, is_author: bool, posts: Vec<Post>)
+
+@:base(ctx, blog.title.as_ref(), {}, {
+    <a href="/~/@fqn">@blog.title</a>
+}, {
+    <h1>@blog.title <small>~@fqn</small></h1>
+    <p>@blog.summary</p>
+    <p>
+        @i18n!(ctx.1, "One author in this blog: ", "{0} authors in this blog: ", authors.len())
+        @for author in authors {
+            <a class="author" href="/@@/@author.get_fqn(ctx.0)">author.name(ctx.0)</a>
+        }
+    </p>
+    <p>
+        @i18n!(ctx.1, "One article in this blog", "{0} articles in this blog", total_articles)
+    </p>
+
+    <section>
+        <h2>
+            @i18n!(ctx.1, "Latest articles")
+            <small><a href="/~/@fqn/atom.xml" title="Atom feed">@icon!("rss")</a></small>
+        </h2>
+        @if posts.len() < 1 {
+            <p>@i18n!(ctx.1, "No posts to see here yet.")</p>
+        }
+        @if is_author {
+            <a href="/~/@fqn/new/" class="button inline-block">@i18n!(ctx.1, "New article")</a>
```

Comment:
I'm not sure why yet, but having a user&display name of `" content=` shift the button to the left

Reply:
this was only on user name (fqn), fixed by 9714bafded2aefeaf2eb4123a338b93c4cd019d8 then